### PR TITLE
feat: add more fun commands

### PR DESCRIPTION
**Pull request by thenishantsapkota**
This command imitates other users using Discord Webhooks

### CODE REVIEW

File: src/commands/Fun/FakeMessageCommand.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    async execute(interaction: CommandInteraction) {
+        const user = interaction.options.getUser("user");
+        if (!interaction.memberPermissions?.has(Permissions.FLAGS.MANAGE_WEBHOOKS)) {
+            return await interaction.reply({ content: "You don't have appropriate permissions to run this command." });
+        }
+        const message = interaction.options.getString("message");
+        // @ts-expect-error TS is weird
+        const channel: TextChannel = await interaction.guild?.channels.fetch(interaction.channelId);
+        const webhook = await channel.createWebhook(user!.username, {
+            avatar: user?.avatarURL(),
```
Comment (DevAndromeda) [inline src/commands/Fun/FakeMessageCommand.ts:24]: we should use `user?.displayAvatarURL()` here

**Review by DevAndromeda — CHANGES_REQUESTED** [review]

File: src/commands/Fun/FakeMessageCommand.ts (hunk 1 above)
Comment (thenishantsapkota) [reply src/commands/Fun/FakeMessageCommand.ts:24]: Got it!

Comment (DevAndromeda) [reply src/commands/Fun/FakeMessageCommand.ts:24]: even better if we add a check for guild avatar and nickname

**Review by DevAndromeda — APPROVED** [review]
looks good

**Pull request merged**